infrastructure-playbooks/switch-from-non-containerized-to-containerized-ceph-daemons.yml

```
@@ -91,13 +91,8 @@
       when: ceph_docker_image is search("rhceph")
 
     - name: set proper ownership on ceph directories
-      file:
-        path: "{{ item }}"
-        owner: "{{ ceph_uid }}"
-        recurse: yes
-      with_items:
-        - /var/lib/ceph
-        - /etc/ceph
+      command: "chown -R {{ ceph_uid }} /var/lib/ceph /etc/ceph"
```

```suggestion
      command: "chown --verbose -R {{ ceph_uid }} /var/lib/ceph /etc/ceph"
```

Hopefully, this won't add too extra time to the command, but could be handy for debugging/logging.

---

I found that adding verbose flag to a test playbook caused errors in the command run as seen below.

TASK [set proper ownership on ceph directories (raw chown verbose)] ***************************************************************************************************************************************************************************
Monday 19 August 2019  11:23:38 -0400 (0:00:12.283)       0:00:12.530 ********* 
ERROR! A worker was found in a dead state

---

This is surprising.